unix/xserver/hw/vnc/vncExtInit.cc

```diff
@@ -184,15 +184,15 @@ void vncExtensionInit(void)
             listeners.push_back(new network::TcpListener(vncInetdSock));
             vlog.info("inetd wait");
           }
-        } else if (rfbunixpath.getValueStr()[0] != '\0') {
+        } else if (rfbUnixPath[0] != '\0') {
```

Comment:
In these cases you can use `operator const char*()` which gives a reference, so no need to be freed.